README.md

```diff
@@ -72,16 +89,16 @@ https://openboxes.atlassian.net/wiki/spaces/OBW/pages/1719435265/Push-button+dep
 ### Install Dependencies
 
 #### Required
-* [Java 7 (must install Java 7)](https://www.azul.com/downloads/?version=java-7-lts&package=jdk)
-* [MySQL 5.7](https://downloads.mysql.com/archives/community/)
+* [Java 8 (must install Java 8)](https://www.oracle.com/pl/java/technologies/javase/javase8-archive-downloads.html) or via SDK
+* [MySQL 5.7](https://downloads.mysql.com/archives/community/) or [MariaDB 10.11.4](https://mariadb.com/kb/en/mariadb-10-11-4-release-notes/)
   * Mac users: 5.7.31 is the latest 5.7.x with a pre-built installer and works fine
 * [SDK Man](https://sdkman.io/install)
-* [Grails 1.3.9](https://grails.org/download.html)
+* [Grails 3.3.10](https://grails.org/download.html)
```

Comment:
Technically it should be 3.3.17 now. Although in the `gradle.properties` we have `3.3.16`, however, this version is not present on the list of older grails versions on the download page. So I think we should have `3.3.17` here.

Reply:
:+1: